src/Controller/TokenController.php

```diff
@@ -80,4 +81,35 @@ public function getToken(string $uniqueId): JsonResponse
 
         return new JsonResponse(['token' => $token->getToken(), 'expire' => $token->getTokenExpires()]);
     }
+
+    /**
+     * @Route("/token/refresh", name="refresh_token", methods={"POST"})
+     *
+     * @param request $request
+     *   The request
+     *
+     * @return JsonResponse
```

Comment:
What is returned? Format of the json maybe?

Reply:
Fixed